j/jszip/jszip_ubi8.sh

```diff
@@ -38,4 +40,5 @@ git clone https://github.com/Stuk/jszip
 cd jszip
 git checkout v$PACKAGE_VERSION
 npm install
-npm test
+# uncomment the below line to run tests
+#npm test
```

Comment:
@vaishnavipatil3 why are we commenting out the tests?

Reply:
Actually npm test command was not terminating so exceeding time limit. So I had commented it.
But found in jszip documentation that -To test JSZip in nodejs, use npm run test-node
So will update and commit the changes